core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -83,10 +83,28 @@
                 </div>
               </div>
             </div>
-            <div class="donate-our-learners">
-              <!--Add the learners heading and carousel here-->
-              <p></p>
-              <div class="learner-carousel"></div>
+            <div class="donate-learners-container">
+              <p> {{ 'I18N_DONATE_PAGE_CONTENT_HEADING_4' | translate }} </p>
+              <div class="donate-learners-bg">
+                <div class="donate-learners-carousel">
+                  <div class="donate-learners-tile" *ngFor="let learner of learners;index as id" id="tile-{{id}}">
```

Comment:
We try not to use IDs since they generally aren't guaranteed to be globally unique within the site, especially since pages tend to get built up using multiple components.

If this is for tests, can you use classes instead? Or maybe find some other way to handle it that doesn't require setting an id on the element?